cmd/vela-npm/main.go

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/go-vela/vela-npm/internal/npm"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:      "vela-npm",
+		Usage:     "Vela npm plugin for publishing NodeJS packages",
+		Version:   "2.1.0",
```

Comment:
version becomes:
``
Version:  pluginVersion.Semantic(),
``

Reply:
I don't have any experience with the version library the other plugins are referencing, but used Kaniko as an example to update this one.